Dockerfile

```
@@ -1,22 +1,21 @@
+FROM openjdk:11-jdk AS build
+# build jar
+ADD . /code
+WORKDIR /code
+RUN ./gradlew clean build --stacktrace
+
 FROM ubuntu:20.04
 
 RUN apt-get update && \
 apt-get install -y --no-install-recommends \
         openjdk-11-jdk
```

For the final image you should only need the JRE and not the full JDK.
```suggestion
        openjdk-11-jre
```